free/src/main/scala/cats/free/Free.scala

```diff
@@ -138,7 +132,7 @@ sealed abstract class Free[S[_], A] extends Product with Serializable {
    * enough, this will work
    */
   final def foldMapUnsafe[M[_]](f: FunctionK[S, M])(implicit M: Monad[M]): M[A] =
-    foldMap[M](f)(M, RecursiveTailRecM.create)
+    foldMap[M](f)
```

Comment:
this should just be removed if you ask me. I doubt anyone is using it and removing `RecursiveTailRecM` is at least as big of an issue.